docs/downloads/automation-library/standard/review_changelog.cm

```diff
@@ -0,0 +1,21 @@
+# -*- mode: yaml -*-
+manifest:
+  version: 1.0
+automations:
+  # Request changes for new features that lack changelog updates.
+  review_changelog: 
+    if: 
+      - {{ branch.name | includes(term="feature") }}
+      - {{ files | match(regex=r/^docs\/changelog\.md$/) | some | nope }}
```

Comment:
this should be just `nope` instead of `some | nope`

Reply:
Great catch. I second-guessed myself and added the some right at the end before submitting the PR :-) It's fixed.